quinn/src/tests.rs

```diff
@@ -523,6 +523,7 @@ fn run_echo(args: EchoArgs) {
             // requires modifying this test - please update the list of supported
             // platforms in the doc comment of `quinn_udp::RecvMeta::dst_ip`.
```

Comment:
Note that Android has already been added here:

https://github.com/quinn-rs/quinn/blob/169f0c2b4380fa7fc4923d9a7a3058f4bc8939d2/quinn-udp/src/lib.rs#L91-L93